pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -69,59 +69,66 @@ def __init__(self, *args, **kargs):
-                             and max of the original data. This method produces the best 
-                             visual representation of the data but is slower.
-            autoDownsample   (bool) If True, resample the data before plotting to avoid plotting
-                             multiple line segments per pixel. This can improve performance when
-                             viewing very high-density data, but increases the initial overhead 
-                             and memory usage.
-            clipToView       (bool) If True, only plot data that is visible within the X range of
-                             the containing ViewBox. This can improve performance when plotting
-                             very large data sets where only a fraction of the data is visible
-                             at any time.
-            identical        *deprecated*
-            ================ =====================================================================
-        
+
+            ================= =====================================================================
+            antialias         (bool) By default, antialiasing is disabled to improve performance.
+                              Note that in some cases (in particluar, when pxMode=True), points
+                              will be rendered antialiased even if this is set to False.
+            decimate          deprecated.
+            downsample        (int) Reduce the number of samples displayed by this value
+            downsampleMethod  'subsample': Downsample by taking the first of N samples.
+                              This method is fastest and least accurate.
+                              'mean': Downsample by taking the mean of N samples.
+                              'peak': Downsample by drawing a saw wave that follows the min
+                              and max of the original data. This method produces the best
+                              visual representation of the data but is slower.
+            autoDownsample    (bool) If True, resample the data before plotting to avoid plotting
+                              multiple line segments per pixel. This can improve performance when
+                              viewing very high-density data, but increases the initial overhead
+                              and memory usage.
+            clipToView        (bool) If True, only plot data that is visible within the X range of
+                              the containing ViewBox. This can improve performance when plotting
+                              very large data sets where only a fraction of the data is visible
+                              at any time.
+            dynamicRangeLimit (float or None) Limit off-screen positions of data points at large
+                              magnification to avoids display errors. Disabled if None.
+        =============== =============================================================
```

Comment:
Looks like you need to indent this line over another 4 spaces (as well as the next couple of lines)